src/main/java/org/spongepowered/api/keyboard/KeyBinding.java

```diff
@@ -96,6 +98,34 @@ static Builder builder() {
         Builder displayName(Text displayName);
 
         /**
+         * Sets the press executor of this key binding, it will be called
+         * when a specific {@link Player} the key binding pressed.
+         *
+         * @param executor The press executor
+         * @return This builder for chaining
+         */
+        Builder pressExecutor(@Nullable BiConsumer<Player, KeyBinding> executor);
+
+        /**
+         * Sets the release executor of this key binding, it will be called
+         * when a specific {@link Player} the key binding release.
+         *
+         * @param executor The release executor
+         * @return This builder for chaining
+         */
+        Builder releaseExecutor(@Nullable BiConsumer<Player, KeyBinding> executor);
+
+        /**
+         * Sets the tick executor of this key binding, it will be called
+         * when a specific {@link Player} the key binding holds the key
+         * and a tick occurred.
+         *
+         * @param executor The tick executor
+         * @return This builder for chaining
+         */
+        Builder tickExecutor(@Nullable BiConsumer<Player, KeyBinding> executor);
```

Comment:
@Cybermaxke, the `CommandSpec` builder does not accept null as an executor, but does internally default to null if you don't specify one.

Unless there's child commands, it will throw an exception if there's a null executor. In this scenario I think it would be acceptable to no-op for null executors -- any or all of the three.